website/docs/r/app_service_certificate.html.markdown

```diff
@@ -0,0 +1,75 @@
+---
```

Comment:
can we add this new page [to the sidebar](https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/website/azurerm.erb#L485-L503)?